Aztec/Classes/TextKit/TextView.swift

```diff
@@ -381,6 +381,11 @@ open class TextView: UITextView {
         pasteboard.items[0][NSAttributedString.pastesboardUTI] = data
     }
 
+    fileprivate func textViewDidChange() {
```

Comment:
Looks good! but perhaps a name like `notifyTextViewDidChange` would work better? (that way it's crystal clear what the method does)